lib/web/apiserver.go

```diff
@@ -282,6 +283,30 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 			return
 		}
 
+		// check proxy and auth node versions (breaking change from v5).
+		// If there is a mismatch with the current proxy client, redirect users to an error page to update proxy.
+		//
+		// TODO remove this block after shipping v4.3
```

Comment:
We have a standard for marking something as being temporary.  I think in this case you're gonna want something like this:

```golang
// DELETE IN 5.0:  <reason-no-longer-needed>
```

That way we can easily grep for all scheduled removals when preparing the `5.0` release.

Also, if you are going to leave a TODO (not needed here, just use the DELETE IN directive) then annotate it with your username, like so:

```golang
// TODO(kimlisa): <thing-that-needs-doing>
```

Reply:
revised